CMakeLists.txt

```diff
@@ -47,7 +47,7 @@ endif()
 set(ARK_COMPILATION_OPTIONS ${CMAKE_CXX_FLAGS})
 set(ARK_COMPILER            ${CMAKE_CXX_COMPILER_ID})
 
-set(ARK_BUILD_EXE       Off CACHE BOOL "Build a standalone ark executable")
+set(ARK_BUILD_EXE       On CACHE BOOL "Build a standalone ark executable")
```

Comment:
Be careful with CMakeLists modifications, in VSCode you can edit the CMake cache through the UI to enable/disable options:
![image](https://user-images.githubusercontent.com/10694085/116827846-8d124f00-ab9b-11eb-900d-6f3cc7dfe382.png)
![image](https://user-images.githubusercontent.com/10694085/116827849-969bb700-ab9b-11eb-891a-197564cd6df5.png)